.env.development

```diff
@@ -2,7 +2,7 @@ REACT_APP_CHAIN_ID = "56"
 REACT_APP_GTAG = "GTM-PXLD3XW"
 
 # 10+ nodes balanced, US/EU
-REACT_APP_NODE_1 = "https://bsc-dataseed1.ninicoin.io"
+REACT_APP_NODE_1 = "https://bsc-dataseed1.binance.io"
```

Comment:
Why did you change the node?

Reply:
I changed it because I'm unable to get response from ninicoin. I will revert it back